### ISSUE TITLE
Allow configurable model-level salt independent of table_name

### ISSUE THREAD

**Issue by jcypret** [jcypret/hashid-rails#60]
The fact that the model name is incorporated into the Hashids config is the main reason I rolled my own instead of using this. I'd love to see this so that I could remove some custom code, though I would encourage not tying this configuration concept to "table" and instead just generally allow a different "seed"/"salt" per-model to be mixed in.

My own version of this gem allows for something like:

``` ruby
class Author < ApplicationRecord
  # `salt`, `alphabet`, etc fallback to global but `pepper` is required
  hashes_ids pepper: 'authors'
end
```

_Originally posted by @bjeanes in https://github.com/jcypret/hashid-rails/pull/47#issuecomment-437730438_

**Comment by bjeanes** [comment]
🙌

For context,i was at the time sure i would be renaming both these tables and the model names and wanted to ensure IDs wouldn't change as a result.

**Comment by jcypret** [comment]
Released a new version of the gem (1.3.0) that supports model-level config (#67). Implemented basically the proposed above but with a slight different method name (`hashid_config`). Let me know if you have any thoughts or feedback @bjeanes 👋. And many thanks again for your input on this feature 🙏
